testlib/test_cases/README.md

```diff
@@ -0,0 +1 @@
+This directory should contain test cases generated from kickstart test repository using scripts/generate-testcases.py script.
```

Comment:
I wonder, why we need to generate them and have them part of the repository. To me it seems like something which should be abstracted. One way we could go is to use their format and really have that as first-class citizen of KS tests.  Another way seems to me would be to not have these completely and generate them when required.

Honestly, I'm more inclined of start using them here as first-class citizen. For example we might migrate our TAGs to this solution and finally have a metadata for tests?

Reply:
That is a follow-up goal I am planning to consider. But as a next isolated step. Moving the metadata will have consequences for various other usages so we need to be careful here,  for example: I want to run all network tests locally for which we need some tooling and easy local usage of permian/tclib (RTT is working on their packaging).  And yes, tags seem as the first step to me.
The test cases (test library with generated test plans and test cases) are part of repository now as a kind of placeholder for future migration.